src/main/java/org/rumbledb/cli/JsoniqQueryExecutor.java

```diff
@@ -108,79 +101,24 @@ public void runLocal(String queryFile, String outputPath) throws IOException {
             String output = runIterators(result, dynamicContext);
             if (outputPath != null) {
                 List<String> lines = Arrays.asList(output);
-                org.apache.commons.io.FileUtils.writeLines(outputFile, "UTF-8", lines);
+                FileSystemUtil.write(outputPath, lines, metadata);
             } else {
                 System.out.println(output);
             }
         }
-        long endTime = System.currentTimeMillis();
-        long totalTime = endTime - startTime;
-        if (this.configuration.getLogPath() != null) {
-            writeTimeLog(totalTime);
-        }
-    }
 
-    public void run(String queryFile, String outputPath) throws IOException {
-        JsoniqLexer lexer = getInputSource(queryFile);
-        long startTime = System.currentTimeMillis();
-        MainModule mainModule = this.parse(lexer);
-        generateStaticContext(mainModule);
-        DynamicContext dynamicContext = VisitorHelpers.createDynamicContext(mainModule, this.configuration);
-        RuntimeIterator result = generateRuntimeIterators(mainModule);
-        // collect output in memory and write to filesystem from java
-        if (this.useLocalOutputLog) {
-            String output = runIterators(result, dynamicContext);
-            org.apache.hadoop.fs.FileSystem fileSystem = org.apache.hadoop.fs.FileSystem
-                .get(SparkSessionManager.getInstance().getJavaSparkContext().hadoopConfiguration());
-            FSDataOutputStream fsDataOutputStream = fileSystem.create(new Path(outputPath));
-            BufferedOutputStream stream = new BufferedOutputStream(fsDataOutputStream);
-            stream.write(output.getBytes());
-            stream.close();
-            // else write from Spark RDD
-        } else {
-            if (!result.isRDD()) {
-                throw new OurBadException("Could not find any RDD iterators in executor");
-            }
-            JavaRDD<Item> rdd = result.getRDD(new DynamicContext());
-            JavaRDD<String> output = rdd.map(o -> o.serialize());
-            output.saveAsTextFile(outputPath);
-        }
         long endTime = System.currentTimeMillis();
         long totalTime = endTime - startTime;
-        if (this.configuration.getLogPath() != null) {
-            writeTimeLog(totalTime);
+        String logPath = this.configuration.getLogPath();
+        if (logPath != null) {
+            String time = "[ExecTime] " + totalTime;
```

Comment:
The other execution time message has been refactored to:
`output("The query took " + time + " milliseconds to execute.");`

Was it intentional to leave this one as Exectime?